### PR TITLE
Identity

### PR DESCRIPTION
**Pull request by Hugo-Hontans**
- [x] Fix identity page
- [x] New design
- [x] Fix terms-conditions

### CODE REVIEW

File: libs/auth/src/lib/auth.module.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ import { MatIconModule } from '@angular/material/icon';
 import { MatBadgeModule } from '@angular/material/badge';
 import { MatTooltipModule } from '@angular/material/tooltip';
 import { MatCheckboxModule } from '@angular/material/checkbox';
+import { MatCardModule } from '@angular/material';
```
Comment (Maxstgt) [inline libs/auth/src/lib/auth.module.ts]: precise import

File: libs/auth/src/lib/pages/identity/identity.component.ts [hunk 2]
```diff
@@ -55,9 +43,8 @@ export class IdentityComponent {
       await this.service.update({
         name: this.form.get('name').value,
         surname: this.form.get('surname').value,
-        avatar: this.form.get('avatar').value
       });
-      this.router.navigateByUrl('/');
+      this.router.navigate(['/c']);
```
Comment (Maxstgt) [inline libs/auth/src/lib/pages/identity/identity.component.ts]: maybe add {relativeTo: this.route}  nad in the constructor ```
private route: ActivatedSnapshot
```

File: libs/auth/src/lib/pages/identity/identity.component.html [hunk 3]
```diff
@@ -1,15 +1,8 @@
 <section fxLayout="column" fxLayoutAlign="center center">
-  <article fxLayout="row" fxLayoutAlign="center center" fxLayout.lt-md="column">
-    <img src="/assets/images/dark-rocket.png">
+  <h1>Join your organization</h1>
+  <img asset="join_orga.svg" alt="Join Organization Illustration"/>
```
Comment (Maxstgt) [inline libs/auth/src/lib/pages/identity/identity.component.html]: maybe add this icons in the icon component as well, so we can use it also as icon...?

Comment (Hugo-Hontans) [reply libs/auth/src/lib/pages/identity/identity.component.html]: No it's an illustration in svg.

**Pull request merged**